eth/filters/filter_system.go

```diff
@@ -371,7 +374,7 @@ func (es *EventSystem) SubscribeNewHeads(headers chan *types.Header) *Subscripti
 
 // SubscribePendingTxs creates a subscription that writes transactions for
 // transactions that enter the transaction pool.
-func (es *EventSystem) SubscribePendingTxs(txs chan []*types.Transaction) *Subscription {
+func (es *EventSystem) SubscribePendingTxs(txs chan []*ethapi.RPCTransaction) *Subscription {
```

Comment:
I feel that at this point - internally - we should not shuffle RPCTransaction objects. It's fine to convert to these types at the last step before returning to the user, but it's an RPC output nicety, Geth internally should not pass around RPC related types.